srcpkgs/gkrellm/template

```diff
@@ -1,18 +1,17 @@
 # Template file for 'gkrellm'
 pkgname=gkrellm
 version=2.3.11
-revision=3
+revision=4
 build_style=gnu-makefile
 hostmakedepends="pkg-config gettext which"
 makedepends="openssl-devel libSM-devel gtk+-devel libsensors-devel"
 short_desc="GTK2 based system monitor"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Benjamín Albiñana <benalb@gmail.com>"
 license="GPL-3.0.or-later"
 homepage="http://gkrellm.srcbox.net"
 distfiles="${homepage}/releases/gkrellm-${version}.tar.bz2"
 checksum=1ee0643ed9ed99f88c1504c89d9ccb20780cf29319c904b68e80a8e7c8678c06
 
 do_install() {
 	make PREFIX=/usr DESTDIR=${DESTDIR} STRIP= install
-	rm -r ${DESTDIR}/usr/{include,lib}
```

Comment:
Should these maybe be pushed onto a `-devel` package?

Do you think the split is worth it, in this case?

Reply:
The only distros I see with gkrellm-devel are Fedora or Centos, Debian, Arch and Gentoo have the includes in the gkrellm package. I don't know is worth it.

Reply:
not worth it